lib/twitter/creatable.rb

```diff
@@ -9,7 +9,10 @@ module Creatable
     #
     # @return [Time]
     def created_at
-      Time.parse(@attrs[:created_at]).utc unless @attrs[:created_at].nil?
+      time = @attrs[:created_at]
+      return if time.nil?
+      return time.utc if time.is_a?(Time)
+      Time.parse(time).utc
```

Comment:
I also could have done ```def created_at
      time = @attrs[:created_at]
      return if time.nil?
      time = Time.parse(time) unless time.is_a?(Time)
      time.utc
    end``` if you prefer

Reply:
Ah, yes. I like this even better. Do you mind making this change. I think this is the last step before merging into master and releasing. Thanks again for all your work on this!